pipeline/main.tf

```diff
@@ -24,7 +24,7 @@ resource "aws_iam_role" "lambda" {
     }]
     Version = "2012-10-17"
   })
-  managed_policy_arns = ["arn:aws:iam::aws:policy/AWSLambda_FullAccess", "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"]
+  managed_policy_arns = ["arn:aws:iam::aws:policy/AWSLambda_FullAccess", "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole",  "arn:aws:iam::aws:policy/AmazonS3FullAccess"]
```

Comment:
Good catch. Well done for fixing the lambda. Lets keep our eyes out for a more limited permission as we go forward.